tools/update-eslint.sh

```diff
@@ -20,14 +20,16 @@ rm -rf node_modules/eslint
         cd node_modules/eslint || exit
 
         npm install --no-bin-links --production --no-package-lock eslint-plugin-markdown@latest
+        cd ../.. || exit
```

Comment:
The `update-eslint.sh` script was broken in this [commit]( https://github.com/nodejs/node/commit/0e96dc1f95be48a70de0beaef20b56cbb510365c#diff-8172ae0db9029ad4b327a2bed4eb2f054d15a1ef559266596d4ea760e5898acd). /cc @aduh95